pkg/openthread/contrib/netdev/openthread_netdev2.c

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
this file is still called `openthread_netdev2.c`. Please rename it `openthread_netdev.c`